config/routes.rb

```diff
@@ -1,8 +1,20 @@
 Rails.application.routes.draw do
+  get 'password_resets/create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.